ir/src/main/scala/org/scalajs/ir/Serializers.scala

```diff
@@ -106,14 +106,16 @@ object Serializers {
     final val FormatNoPositionValue = -1
   }
 
-  private final class ByteString(val bytes: Array[Byte]) {
+  private final class EncodedNameKey(val encoded: EncodedName) {
```

Comment:
If `EncodedName` gets the equals and hash code directly, this class can simply disappear, right?

Reply:
Yes, but as I said above, `EncodedName` cannot get `equals` and `hashCode` :(